dials_data/definitions/MyD88_processed.yml

```diff
@@ -0,0 +1,13 @@
+name: MyD88 processed
```

Comment:
This is a human readable name, so... what is `MyD88`, why is it `processed`, and what does it mean?